src/equations/bbm_bbm_1d.jl

```diff
@@ -53,6 +53,41 @@ function initial_condition_convergence_test(x, t, equations::BBMBBMEquations1D,
     return SVector(eta, v)
 end
 
+"""
+    initial_condition_manufactured(x, t, equations::BBMBBMEquations1D, mesh)
+
+A smooth manufactured solution in combination with `source_terms_manufactured`.
```

Comment:
```suggestion
A smooth manufactured solution in combination with [`source_terms_manufactured`](@ref).
```
To get links in the docs